src/Controller/FileController.php

```diff
@@ -5,18 +5,47 @@
 use App\Repository\PhotoRepository;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Session\Session;
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
 use Symfony\Component\Routing\Annotation\Route;
 
+use function PHPUnit\Framework\throwException;
+
 class FileController extends AbstractController
 {
-    #[Route('/file/{title}', 'app_file')]
-    public function example(string $title, PhotoRepository $photoRepository): Response
+    #[Route('/file/{id}', 'app_file')]
+    public function getFile(int $id, PhotoRepository $photoRepository, Session $session): Response
```

Comment:
Avec Symfony tu as un ParamConverter doctrine qui te récupère directement l'entité :
```suggestion
    #[Route('/file/{photo}', 'app_file')]
    public function getFile(Photo $photo, Session $session): Response
```